apps/studio/src/lib/db/clients/sqlserver.ts

```diff
@@ -692,7 +693,7 @@ export class SQLServerClient extends BasicDatabaseClient<SQLServerResult> {
     return {
       totalRows: undefined,
       columns: undefined,
-      cursor: new SqlServerCursor(this.connection, query, chunkSize),
+      cursor: new SqlServerCursor({ dbConfig: this.dbConfig }, query, chunkSize),
```

Comment:
@not-night-but The cursor shouldn't make it's own new connection, it should take the existing `pool` and use that.